app/core/components/FormInput/FormInput.styles.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled"
+
+export const PersonalizaedFormInput = styled.div`
```

Comment:
```suggestion
export const PersonalizedFormInput = styled.div`
```

And probably fixed in other places as well 🙈 . But I wonder if this could be done with other properties for MUI components instead of custom styles.

Most are `visibility: hidden`. For an input, maybe you can use an <input type=hidden>` instead. Why are we using them? 🙈.

About the flex direction, why are we using it? Because there are other components that do the same, like [Stack](https://mui.com/material-ui/api/stack/) or [Grid](https://mui.com/material-ui/api/grid/) that have a `direction` property.

Reply:
I mean using a Grid or Stack component instead of custom CSS